v2/ticker_service.go

```diff
@@ -70,6 +72,8 @@ func (s *ListPricesService) Do(ctx context.Context, opts ...RequestOption) (res
 	}
 	if s.symbol != nil {
 		r.setParam("symbol", *s.symbol)
+	} else if s.symbols != nil {
+		r.setParam("symbols", *s.symbols)
```

Comment:
Do not manually concatenate substrings to json, since `symbols` is `[]string`, we can just use `json.Marshal`:
```suggestion
                s, _ := json.Marshal(s.symbols)
		r.setParam("symbols", s)
```

Reply:
Ok, just had to convert to string before setting the param, so that the test pass:
Requested:
`                s, _ := json.Marshal(s.symbols)
		r.setParam("symbols", s)`
Actual:
`		s, _ := json.Marshal(s.symbols)
		r.setParam("symbols", string(s))`
I mention it because it could be that my beginner knowledge of Go overlooks something simpler.